pkg/plugins/hybrid/v1alpha/scaffolds/internal/templates/makefile.go

```diff
@@ -111,8 +111,11 @@ help: ## Display this help.
 
 ##@ Build
 
+build: generate fmt vet ## Build manager binary.
```

Comment:
It would make sense to define a `.PHONY` target for build as in here: (https://github.com/operator-framework/operator-sdk/blob/3af4acbea7efdeac0db146937fb6be10a8983f41/testdata/go/v4-alpha/memcached-operator/Makefile#L113). The existing `all` calls a `build`, and this would ensure we call the build recipe.